GVFS/GVFS.Platform.Mac/MacPlatform.Shared.cs

```diff
@@ -15,7 +17,15 @@ public static bool IsElevatedImplementation()
 
         public static bool IsProcessActiveImplementation(int processId)
         {
-            // TODO(Mac): Implement proper check
+            try
+            {
+                Process process = Process.GetProcessById(processId);
```

Comment:
Do we need to check that the `process` is not null or match the processId that we passed in?  Is an exception the only indication that the process is not active?

Reply:
No need to check `process`. We already follow this pattern in  `WindowsPlatform.Shared::IsProcessActiveImplementation` and also in `GitReadAndGitLockTests::OrphanedGVFSLockIsCleanedUp()`)